client/dist/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8" />
     <link rel="stylesheet" type="css" href="../styles.css"/>
     <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
-  <script defer src="booking.js?c2b519fc37c92ffb64e5"></script></head>
+  <script defer src="booking.js?15eb6958c0b7f85d502f"></script></head>
```

Comment:
I know you're working out your gut ignore file, but this would be a good thing to add to it! 😉

Reply:
Oh no, I thought I'd worked this out by ignoring my ist folder. Ok, thx for catching. I need to look into why it is still not being ignored.